.env

```diff
@@ -0,0 +1,5 @@
+SERVER_PUBLIC_DOMAIN="http://didcomm-mediator.com"
+SERVER_LOCAL_PORT="8080"
+STORAGE_DIRPATH="crates/generic-server/target/storage"
+MONGO_DBN="mediator-coordination"
```

Comment:
Why choose 'mediator-coordination' as the database name? A straightforward name like 'mongodb' might make more sense.